### PR TITLE
Simple Doc Updates

### PR DESCRIPTION
**Pull request by alexpaden**
I've made some simple changes to the docs based on my experience reading them. 
This does not include new examples.

---

## PR-Codex overview
This PR updates the README.md file with Yarn commands, fixes a broken link, adds a disclaimer to the CPU and storage requirements, and provides additional resources for developers. It also adds information about the difference between Farcaster and Warpcast in the FAQ.

### Detailed summary
- Replaced `npm` commands with `yarn` commands in README.md
- Fixed a broken link in `docs/protocol/overview.md`
- Added a disclaimer to the CPU and storage requirements in `docs/developers/hubs.md`
- Added a community-maintained list of resources in `docs/developers/community.md`
- Added information about the difference between Farcaster and Warpcast in `docs/intro/faq.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: docs/intro/faq.md [hunk 1]
```diff
@@ -13,8 +13,13 @@ Farcaster's identity layer is currently on the Goerli Ethereum testnet. We plan
...
 
 ## Why is Farcaster invite only?
+
 For now, invites are limited to minimize spam. If you're a person who actively engages on social networks or writes code, you shouldn't have a problem getting an invite. We're working on opening the system so that anyone can register by the end of the year.
 
 ## Where can I find a public roadmap?
 
 You can find a public roadmap [here](https://hackmd.io/@farcasterxyz/SyeXtT4fh).
+
+## What is the difference between Farcaster & Warpcast
```
Comment (varunsrin) [inline docs/intro/faq.md:23]: How about something like: 

```
Farcaster is an open protocol (like email), while Warpcast is one of many specific apps (like Gmail) that implements the protocol. 
```

File: docs/developers/hubs.md [hunk 2]
```diff
@@ -20,5 +20,10 @@ A Hub can be run on any machine that has:
 - 8 GB of RAM
 - 2 CPU cores or vCPUs
 - 20 GB of free storage
+- An Ethereum Goerli node RPC URL
```
Comment (varunsrin) [inline docs/developers/hubs.md:23]: nit: `Access to an Ethereum Goerli node via an RPC URL`

File: docs/developers/community.md [hunk 3]
```diff
@@ -22,6 +22,17 @@ Github is where the work gets done. It's the best place for technical conversati
 | [Contracts](https://github.com/farcasterxyz/contracts) | An implementation of the Farcaster Contracts |
 | [Hubble](https://github.com/farcasterxyz/hub-monorepo) | An implementation of a Farcaster Hub         |
 
+## Community SDKs
+
+This section references the language specific Farcaster community built SDKs for interacting hubs directly, or the warpcast APIs.
+
+| Language                                                      | Description                               |
+| ------------------------------------------------------------- | ----------------------------------------- |
```
Comment (varunsrin) [inline docs/developers/community.md:30]: these are all warpcast api sdks, not farcaster sdks so i think we should remove this section

File: docs/protocol/concepts/identity.md [hunk 4]
```diff
@@ -3,3 +3,11 @@
 A user identity is a numeric identifier like `8930123` controlled by a key pair. Farcaster uses a smart contract registry on a Turing-complete blockchain to map identifiers to key pairs. Users can generate a new key pair or address and register an identity with the contract. The registry allows key rotation in case of exposure, and smart contract wallets can protect against key loss.
 
 Identifiers, known as **Farcaster IDs** or **fids**, are numeric values that are cheap, meaningless, and in unlimited supply. Users can associate human-readable names from namespaces with their fids when using apps. Separating the identity and namespace layers allows identity to remain very decentralized. Problems of trust, like squatting and impersonation, are the domain of the namespace layer. Users are free to choose the namespace they prefer or even use many at once.
+
```
Comment (varunsrin) [inline docs/protocol/concepts/identity.md:6]: this shows up a little weird in the layout. i do agree that we should clarify these terms. 

do you want to remove from this PR and I'll play around with a nice way to add it back?

**Review by varunsrin — CHANGES_REQUESTED** [review]
Thanks for doing this! Left some feedback

File: docs/developers/community.md (hunk 3 above)
Comment (alexpaden) [reply docs/developers/community.md:30]: I tried to tune this around only hubs, but man, it feels so overwhelming sometimes that I felt compelled...

Maybe I'll play around with a link to the awesome-farcaster list

**Comment by alexpaden** [comment]
Alright, this should include all changes. 
I removed the SDK API references but added my open-source project and SDK GitHub in place of it
/alexpaden/farcaster-resources.

**Pull request closed without merging**